stack.rb

```diff
@@ -0,0 +1,28 @@
+class Stack
+
+  def initialize(str)
+    @str = str
+  end
+
+  def pop
+    str2 = ""
+    last_item = @str[@str.length - 1]
+    (@str.length - 1).times { |i| str2[i] = @str[i] }
```

Comment:
Of course, this is so straightforwad. Thank you for pointing all these improvements:)